nailgun/entities.py

```diff
@@ -2863,8 +2863,6 @@ def add(self, synchronous=True, timeout=None, **kwargs):
         if 'data' not in kwargs:
             # data is required
             kwargs['data'] = dict()
-        if 'component_ids' not in kwargs['data']:
```

Comment:
This isn't present in the API anymore for this endpoint ( and it might have never been? not sure if this was ever used in tests. )

Reply:
anyways, as written it makes this endpolnt non-functional for at least the case where you want to use the `latest` flag

Reply:
Can you tell which Sat version removed this attribute? If it is not present in the API, it might be a good idea to cherry-pick thic patch also to the respective `.z` branches.

Reply:
I think that must be 6.14 and I agree that it must be cherrypicked back to the right version!

Reply:
I'll see if I can't walk back a few versions of satellite, and see where this was dropped.